hstream.proto

```diff
@@ -211,7 +211,20 @@ message Stream {
   uint32 shardCount = 4;
 }
 
-message HStreamRecordBatch { repeated bytes batch = 1; }
+enum CompressionType {
+  None = 0;
+  Gzip = 1;
+}
+
+message BatchedRecord {
+  CompressionType compressionType = 1;
+  // Required.
+  // The time at which the message was published,
+  // populated by the server.
+  google.protobuf.Timestamp publish_time = 2;
+  uint64 batchSize = 3;
```

Comment:
What does this `batchSize` do? Can you make some comments about this field?

Reply:
explained in the design document, will add a comment here too

Reply:
fixed